Examples/Modules/laser_injection_from_file/inputs.3d_test_txye

```diff
@@ -0,0 +1,47 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 150
+amr.n_cell =  352 128 128
+amr.plot_int = 150
+amr.max_grid_size = 176 128 128
```

Comment:
This is exactly equivalent to amr.max_grid_size = 176: several values on this line do not mean several directions, they refer to several MR levels (and this run uses level = 0 only). If you want to specify per-direction, the syntax is
```
amr.max_grid_size_x = ...
etc.
```
In your case, it is likely giving the same result as
```suggestion
amr.max_grid_size = 176
```

Reply:
Ah yes! I've learnt that several weeks ago. But this PR is very old and I left this mistake in...